prow/tide/tide.go

```diff
@@ -281,7 +281,12 @@ func (c *Controller) Sync() error {
 		if label := c.ca.Config().Tide.BlockerLabel; label != "" {
 			c.logger.Debugf("Searching for blocking issues (label %q).", label)
 			orgs, repos := c.ca.Config().Tide.Queries.OrgsAndRepos()
-			blocks, err = blockers.FindAll(c.ghc, c.logger, label, orgs, repos)
+			allOrgs := make([]string, 0, len(orgs))
+			for org := range orgs {
```

Comment:
Maybe this is a golang idiom I'm just not used to, but knowing this is a map, I might expect `org, _`